charts/rancher-logging/0.1.2/charts/fluentd/values.yaml

```diff
@@ -0,0 +1,395 @@
+nameOverride: fluentd
+
+rbac:
+  create: true
+
+serviceAccount:
+  create: true
+
+service:
+  type: ClusterIP
+  ports:
+    - name: metric
+      port: 24231
+      targetPort: metric
+
+secrets:
+  entry:
+    fluent.conf: |-
```

Comment:
Can we move all the secret data to other files instead of `values.yaml`

Reply:
Place these configures in value in order to allow user change them, if you place these configures in directly in secret, after redeploy the app, user's configure will be overwrite by default.